IS7024/HBowl.cs

```diff
@@ -480,7 +480,7 @@ public override void WriteJson(JsonWriter writer, object untypedValue, JsonSeria
                 return;
             }
             var value = (MetroAreaDisplayName)untypedValue;
-            if (value == MetroAreaDisplayName.LosAngelesLa)
+            if (value == MetroAreaDisplayName.LosAngeles)
```

Comment:
This change was made so that "LosAngeles" shows up when the "DisplayName" is called for MetroArea.